doc/config/common

```diff
@@ -91,7 +91,7 @@ ALIASES += constants_page{1}="@page \1_constants Constants"
 ALIASES += constants_brief{1}="@brief Defined constants of the \1.<br><br>Libraries may @c #`define` constants in their headers with special meanings. This page describes the meanings and uses of any constants defined by the \1. Related constants are shown in a single section on this page.<br>"
 
 # Alias for starting a functions page.
-ALIASES += functions_page{1}="@page \1_functions Functions"
+ALIASES += functions_page{2}="@page \1_functions Functions"
```

Comment:
Fix for previous PR. Broke functions page, this will make it look normal again. [link](https://imgur.com/a/gzvfBNd)